### PR TITLE
Fix angular-rails-templates version for rubygem-katello

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -56,7 +56,7 @@ Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) < 0.3.0
 Requires: %{?scl_prefix}rubygem(deface) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(deface) < 2.0.0
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 1.0.2
-Requires: %{?scl_prefix}rubygem(angular-rails-templates) < 1.1
+Requires: %{?scl_prefix}rubygem(angular-rails-templates) <= 1.1
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec:59]: Set this to greater than or equal to 1.1

**Comment by jturel** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
Ignore EL8 failures for right now.

**Pull request merged**